app/assets/stylesheets/listings.scss

```diff
@@ -27,3 +27,11 @@
     display: none;
   }
 }
+.tileBrowser-listItem {
+  &.box {
+    margin: 0.2em;
+  }
+  .tagList-tag {
+    margin: 0.1em;
+  }
+}
```

Comment:
I looked at both Tiles and Cards, and I found the number of tweaks I'd have to do in each case more frustrating than what I ended up with. 

Bulma Tiles work great if you know exactly how many of everything you have and where it will go before you start defining the layout. We may need to get there eventually with pagination, but I didn't want to go there yet before it's required

Cards required even more tweaks to get everything to line up the way I wanted for legibility reasons. I ended up using some of the CSS from Cards as my basis here. I had to throw away most of it, though

Reply:
:(

Reply:
Thanks for hangin in there on it, @h-m-m !